modules/fundamental/tests/sbom/issue_1535.rs

```diff
@@ -0,0 +1,20 @@
+use test_context::test_context;
+use test_log::test;
+use trustify_module_ingestor::service::Format;
+use trustify_test_context::TrustifyContext;
+
+/// This is a test for issue #1535
+#[test_context(TrustifyContext)]
+#[test(tokio::test)]
+async fn not_really_clearly_defined(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let result = ctx
+        .ingest_document_as("csaf/timeout/rhsa-2024_5363.json.xz", Format::SBOM)
+        .await;
+
+    if let Err(e) = result {
+        let msg = format!("{e}");
+        assert!(msg.contains("only SPDX and CycloneDX are supported"));
+    }
```

Comment:
The `if let` is confusing here. If `result.is_ok()` then the assert never occurs. Maybe just...
```suggestion
    assert!(result.is_err());
```

Reply:
thanks for this suggestion, I was being very specific grabbing the error message but yeah we don't need that because the focus is the panic situation in case we decide to use serde_yml.